llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2529,6 +2529,13 @@ class VPWidenMemoryRecipe : public VPRecipeBase {
     llvm_unreachable("VPWidenMemoryRecipe should not be instantiated.");
   }
 
+  /// Get element Type
+  Type *getElementType() const { return getLoadStoreType(&Ingredient); }
```

Comment:
Simpler to just use `getLoadStoreType` directly in `::computeCost`, like other `getLoadStoreXXX` helpers there?

Reply:
Thanks, I will directly use the `getLoadStoreType`.